### PR TITLE
Updating the compiler binding redirects to include the Update 3 bits.

### PR DESCRIPTION
**Pull request by tannergooding**
FYI. @dotnet/roslyn-compiler

### CODE REVIEW

**Comment by tannergooding** [comment]
The redirects are now `<bindingRedirect oldVersion="1.0.0.0-2.0.0.0" newVersion="2.0.0.0" />`, which should include all v1 bits. However, should it be `oldVersion="0.0.0.0-2.0.0.0"` (as S.C.Immutable and S.R.Metadata do) to include anything that is pre v2.0, or will that break the bootstrap compiler?

**Comment by tannergooding** [comment]
test vsi please

**Comment by tannergooding** [comment]
Also FYI. @dotnet/roslyn-infrastructure

File: src/Compilers/CSharp/CscCore/csc.exe.config [hunk 1]
```diff
@@ -8,15 +8,15 @@
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
       <dependentAssembly>
         <assemblyIdentity name="Microsoft.CodeAnalysis" publicKeyToken="31bf3856ad364e35" culture="neutral" />
-        <bindingRedirect oldVersion="1.0.0.0-1.65535.65535.65535" newVersion="2.0.0.0" />
+        <bindingRedirect oldVersion="1.0.0.0-2.0.0.0" newVersion="2.0.0.0" />
```
Comment (agocke) [inline src/Compilers/CSharp/CscCore/csc.exe.config]: Just delete this file. The binding redirects are useless on CoreCLR.

File: src/Compilers/VisualBasic/VbcCore/vbc.exe.config [hunk 2]
```diff
@@ -8,15 +8,15 @@
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
       <dependentAssembly>
         <assemblyIdentity name="Microsoft.CodeAnalysis" publicKeyToken="31bf3856ad364e35" culture="neutral" />
-        <bindingRedirect oldVersion="1.0.0.0-1.65535.65535.65535" newVersion="2.0.0.0" />
+        <bindingRedirect oldVersion="1.0.0.0-2.0.0.0" newVersion="2.0.0.0" />
```
Comment (agocke) [inline src/Compilers/VisualBasic/VbcCore/vbc.exe.config]: Same here.

**Comment by agocke** [comment]
LGTM

File: src/Compilers/CSharp/CscCore/csc.exe.config (hunk 1 above)
Comment (tannergooding) [inline src/Compilers/CSharp/CscCore/csc.exe.config]: Removed.

File: src/Compilers/VisualBasic/VbcCore/vbc.exe.config (hunk 2 above)
Comment (tannergooding) [inline src/Compilers/VisualBasic/VbcCore/vbc.exe.config]: Removed.

**Comment by jasonmalinowski** [comment]
:+1:

**Pull request merged**